src/components/FeedsList.jsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function FeedList(props) {
+  function timeSince(date) {
```

Comment:
Since this is a common function, create file name Utils.js and add these functions there so that we can reuse it.